src/components/studies/scheduler/IntroInfo.tsx

```diff
@@ -147,21 +147,17 @@ const IntroInfo: React.FunctionComponent<IntroInfoProps> = ({id, onShowFeedback,
           duration,
         }
       }
-
-      mutateSchedule(
-        {
+      try {
```

Comment:
changing mutate to async with await and try/catch blocks. The redirect only happens when everything is compelted